src/server/controllers/index.js

```diff
@@ -0,0 +1,84 @@
+const  express = require('express');
+var router = express.Router();
+
+/* Require model */
+var models = require('../models');
+/* Require middlewares functions */
+
+router.get('/', function (req, res) {
```

Comment:
I haven't realized about this issue, there are not any difference in both, the difference is in the file api.js, that allows the access for external domains.